cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/CardShowTakenPictureView.java

```diff
@@ -158,12 +159,7 @@ public void checkIfHasImages() {
 
     @Override
     public void showPreviewPicDialog(CardShowTakenImage cardShowTakenImage) {
-        if (cardShowTakenImage.getTempImagePathToShow() != null) {
-            mCardShowTakenPicturePreviewDialogBinding.setImageUrl(cardShowTakenImage.getTempImagePathToShow());
-        } else if (cardShowTakenImage.getLocalImageFilename() != null) {
-            Bitmap bitmap = ImageViewFileUtil.getBitMapFromFile(cardShowTakenImage.getLocalImageFilename(), ImageViewFileUtil.getFile());
-            mCardShowTakenPicturePreviewDialogBinding.previewImage.setImageBitmap(bitmap);
-        }
+        setImageBitmapToImageView(mCardShowTakenPicturePreviewDialogBinding.previewImage, cardShowTakenImage, 1);
```

Comment:
o que seria esse valor 1, se for o caso melhor colocar em uma variavel ou constante descrevendo melhor pra o que o cara não precise ir até a implementacao do metodo pra ver